pontoon/sync/vcs/models.py

```diff
@@ -431,6 +441,53 @@ def resources_for_path(self, path):
                     yield os.path.join(root, filename)
 
 
+class VCSConfiguration(object):
+    """
+    Container for the project configuration, provided by the optional
+    configuration file.
+
+    For more information, see:
+    https://moz-l10n-config.readthedocs.io/en/latest/fileformat.html.
+    """
+
+    def __init__(self, vcs_project):
+        self.vcs_project = vcs_project
+        self.configuration_file = vcs_project.db_project.configuration_file
+
+    @cached_property
+    def parsed_configuration(self):
+        """Return parsed project configuration file."""
+        db_project = self.vcs_project.db_project
+
+        path = os.path.join(
+            db_project.source_repository.checkout_path,
+            self.configuration_file,
+        )
+
+        l10n_base = db_project.translation_repositories()[0].checkout_path
```

Comment:
Source code for [`translation_repositories`](https://github.com/mozilla/pontoon/blob/981114359923d7656f563b696af36c94392283b7/pontoon/base/models.py#L1382) and [`checkout_path`](https://github.com/mozilla/pontoon/blob/981114359923d7656f563b696af36c94392283b7/pontoon/base/models.py#L1669).

Example paths for some projects:
```python
>>> Project.objects.get(slug="firefox").translation_repositories()[0].checkout_path
u'/app/media/projects/firefox/l10n-central'

>>> Project.objects.get(slug="mozillaorg").translation_repositories()[0].checkout_path
u'/app/media/projects/mozillaorg/git@github.com:mozilla-l10n/www.mozilla.org.git'

>>> Project.objects.get(slug="amo").translation_repositories()[0].checkout_path
u'/app/media/projects/amo/git@github.com:mozilla/olympia.git'
```

Reply:
Note: We also have locales coming from bitbucket repositories, for which the checkout path is different.